src/miniscript/types/extra_props.rs

```diff
@@ -476,6 +556,18 @@ impl Property for ExtData {
                 (_, Some(x)) | (Some(x), _) => Some(x + 3),
                 (None, None) => None,
             },
+            stack_elem_count_sat: match (l.stack_elem_count_sat, r.stack_elem_count_sat) {
+                (Some(l), Some(r)) => Some(1 + cmp::max(l, r)),
```

Comment:
Currently, the library only has support for non-malleable scripts. So, it is safe to transform this into unreachable! . In future(I doubt) if we want to support malleable miniscripts, we would need this. 

In any case, no harm in having this.

Reply:
Fun thing is that it makes a lot of tests panic in the current (not correct) state. Will try out after applying all the corrections.

Reply:
Ok, even after the fix it still makes everything panic.. Didn't look into it yet.

Reply:
Sorry, I commented at the wrong place. This should be dissat

Reply:
This should be as it is. https://github.com/rust-bitcoin/rust-miniscript/pull/150/files#r502990941 can be changed

Reply:
Commenting there, resolving this.